### ISSUE TITLE
Average Issue Density includes ignored issues in calculation

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1197]
There are no open issues, only ignored issues on this site, but it still has an issue density:

<img width="1287" height="584" alt="Image" src="https://github.com/user-attachments/assets/bb648eec-bb9c-4e74-9866-0575e0b7bf5e" />

In this case, the issue density should be zero.
